reccmp/isledecomp/compare/core.py

```diff
@@ -313,7 +311,7 @@ def _compare_match(self, match: ReccmpMatch) -> DiffReport | None:
                 is_stub=True,
             )
 
-        if match.entity_type == EntityType.FUNCTION:
+        if match.entity_type in (EntityType.FUNCTION, EntityType.VTORDISP):
```

Comment:
Just curious: What's the reason to not incude thunks here? If there is one, maybe add a comment why